platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/builder/PlatformBuilder.java

```diff
@@ -178,19 +146,16 @@ public final class PlatformBuilder {
      * @param swirldName          the name of the swirld, currently used for deciding where to store states on disk
      * @param selfId              the ID of this node
      * @param softwareVersion     the software version of the application
-     * @param genesisStateBuilder a supplier that will be called to create the genesis state, if necessary
-     * @param snapshotStateReader a function to read an existing state snapshot, if exists
+     * @param initialState        a supplier that will be called to create the genesis state, if necessary
      */
     @NonNull
     public static PlatformBuilder create(
             @NonNull final String appName,
             @NonNull final String swirldName,
             @NonNull final SoftwareVersion softwareVersion,
-            @NonNull final Supplier<MerkleRoot> genesisStateBuilder,
-            @NonNull final CheckedBiFunction<MerkleDataInputStream, Path, MerkleRoot, IOException> snapshotStateReader,
+            @NonNull final ReservedSignedState initialState,
```

Comment:
Is the `initialState` still a supplier? I see that its type has changed to `ReservedSignedState` and the latter name suggests it's a "state", rather than a "state supplier".

Also, it's unclear if the "if necessary" part still applies.

Could you please clarify and/or update the doc to describe what this parameter actually represents after this refactoring?

Same comment applies for line 168 right below.

Reply:
Yes @anthony-swirldslabs , fixed the javadoc. Thanks